backend/src/utils.js

```diff
@@ -0,0 +1,9 @@
+const validator = require('validator');
+
+const sanitize = (message) => {
+    return validator.escape(validator.trim(message));
+};
+
+module.exports = {
+    sanitize,
+};
```

Comment:
Since we have a utils/utils.js directory merged into main where for the functions used for recommendations shall we place this there?

Reply:
Yup :+1: